clients/unshaded/src/main/java/tachyon/client/ReadType.java

```diff
@@ -15,15 +15,17 @@
 
 package tachyon.client;
 
+import javax.annotation.concurrent.ThreadSafe;
+
 import tachyon.annotation.PublicApi;
 
 /**
- * Convenience modes for commonly used read types for a {@link tachyon.client.file.TachyonFile}.
+ * Convenience modes for commonly used read types.
  *
  * For finer grained control over data storage, advanced users may specify
- * {@link tachyon.client.TachyonStorageType} and {@link tachyon.client.UnderStorageType}.
+ * {@link TachyonStorageType} and {@link UnderStorageType}.
  */
-@PublicApi
+@PublicApi @ThreadSafe
```

Comment:
Each annotation should go on its own line https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations. Apply throughout PR